models/business-unit/src/company/company-draft/generator.ts

```diff
@@ -0,0 +1,28 @@
+import { AddressDraft } from '@commercetools-test-data/commons';
+import { fake, Generator, oneOf } from '@commercetools-test-data/core';
+import { status, storeMode, unitType, associateMode } from '../../constants';
+import type { TBusinessUnitDraft } from '../../types';
+
+// https://docs.commercetools.com/api/projects/business-units#businessunitdraft
+
+const generator = Generator<TBusinessUnitDraft>({
+  fields: {
+    key: fake((f) => f.lorem.slug(2)),
+    status: oneOf(...Object.values(status)),
+    stores: [],
+    storeMode: storeMode.Explicit,
```

Comment:
Are you just setting the DivisionDraft specific fields to satisfy the Type definition? Is this modelling the data accurately compared to what we really get in the API?